resources/cdk/cognito_scenario_user_pool_with_mfa/setup.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Defines an AWS CloudFormation stack that creates AWS resources for Amazon Cognito
+ * scenarios. The scripts in this example create the following resources:
+ *
+ * * An Amazon Cognito user pool that is configured to:
```

Comment:
configured to: --> configured to do the following:

Reply:
fixed